### PR TITLE
Style changes doc

### PR DESCRIPTION
**Pull request by tacaswell**

### CODE REVIEW

**Comment by tacaswell** [comment]
I am in the process of re-building the docs to push up

File: doc/style_changes.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+Everything Else
+---------------
+
+We are soliciting proposals to change any and all visual defaults
+(including adding new rcParams as needed).
+
+If you have a proposal please create an issue or PR on `github <https://github.com/matplotlib/matplotlib/issues/new>`_ with the
+changes to `rcsetup.py` and `matplotlibrc.template` by August 9, 2015.
+
+In the second week of August, Micheal Droettboom and I will decide on
```
Comment (jenshnielsen) [inline doc/style_changes.rst]: I guess you copied this from the mail? However this does not have an author so I guess we should put your name here

**Comment by jenshnielsen** [comment]
Apart from the inline comment :+1:

File: doc/style_changes.rst (hunk 1 above)
Comment (efiring) [inline doc/style_changes.rst]: sp: Michael

Comment (tacaswell) [inline doc/style_changes.rst]: good catches, sorry!

**Pull request merged**